subsys/bluetooth/mesh/settings.c

```diff
@@ -2260,6 +2260,7 @@ void bt_mesh_store_cdb_node(const struct bt_mesh_cdb_node *node)
 	}
 
 	free_slot->addr = node->addr;
+	free_slot->clear = false;
```

Comment:
What about the other fields? Valid/Key etc?
Would it make sense to memset it?

Reply:
There are no other fields in this struct actually...

Reply:
Right. different struct. :)